package.json

```diff
@@ -2,7 +2,7 @@
     "name": "vscode-azurevirtualmachines",
     "displayName": "Azure Virtual Machines",
     "description": "%extension.description%",
-    "version": "0.5.1-alpha",
+    "version": "0.5.1-alpha.1",
```

Comment:
Is this our way of distinguishing unified versions from normal versions?

Reply:
To answer my own question (after seeing other PRs): yes this is how we're versioning unified extension changes rn

Reply:
More specifically, this is our to make sure everything you install a new vsix from main (or unified in this case) it will be a higher version than your currently installed one.

If the vsix is the same version as the one you have installed, it won't do anything.